frame/executive/src/lib.rs

```diff
@@ -220,6 +220,58 @@ where
 		weight
 	}
 
+	/// Execute given block, but don't do any of the [`final_checks`].
```

Comment:
We also skip signature verification, correct?

Reply:
Hmm actually considering your attempt at skipping sig verification below, I assume this does verify signatures. Not sure what the significance is of the fact that we dont call `signature_batching.verfiy()` in here tho

Reply:
batch verification is different and generally not used, regardless pf having its code there or not. I am npt sure why it is there while it is not really used in production. 

Normal signature verification always happens (in '.check') and is currently part of the code path.